src/main/java/org/intellij/sdk/codesync/Constants.java

```diff
@@ -65,16 +67,16 @@ private Constants() {
     public static final String WEBSOCKET_ENDPOINT = configuration.getCodeSyncWebsocketURL();
 
     public static final String API_ENDPOINT = String.format("%s/v1", CODESYNC_HOST);
-    public static final String API_INIT = String.format("%s/init", API_ENDPOINT);
-    public static final String API_USERS = String.format("%s/users", API_ENDPOINT);
+    public static final String API_INIT = String.format("%s/init&source=%s&v=%s", API_ENDPOINT, DIFF_SOURCE, PLUGIN_VERSION);
```

Comment:
I think it will go as
```"%s/init?source=%s&v=%s"```
i.e. `?` right after original URL and then `&` separated query params. 
Thinking if there a better way to build a URL using query parameters in Java? @syedgulahmed 
Thoughts @saleem-latif ?

Reply:
e.g. I am using it as: https://github.com/codesyncapp/codesync-vs/pull/81/files#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199R34 in VSCode @syedgulahmed

Reply:
Thank you Basit! I wll discuss it with @saleem-latif